services/header/rocketjoe/services/python_sandbox/detail/celery.hpp

```diff
@@ -1,45 +1,129 @@
 #pragma once
 
+#include <memory>
+#include <string>
+#include <queue>
+#include <unordered_map>
+
 #include <boost/dll.hpp>
+#include <boost/functional/hash.hpp>
+#include <boost/optional.hpp>
+#include <boost/smart_ptr/intrusive_ptr.hpp>
+#include <boost/smart_ptr/intrusive_ref_counter.hpp>
+#include <boost/uuid/uuid.hpp>
 
-#include <pybind11/stl.h>
-#include <pybind11/functional.h>
 #include <pybind11/pybind11.h>
 
 namespace rocketjoe { namespace services { namespace python_sandbox { namespace detail {
 
-            using namespace pybind11::literals;
-            namespace py = pybind11;
-            using py_task = py::function;
+    using namespace pybind11::literals;
+    namespace py = pybind11;
+
+    class celery;
+
+    class BOOST_SYMBOL_VISIBLE result_base {};
```

Comment:
Why do you need a result_base?

Reply:
Because it's in the Celery API: https://docs.celeryproject.org/en/latest/reference/celery.result.html#celery.result.ResultBase